locales/en-US/faq.properties

```diff
@@ -2,38 +2,52 @@
+faq_item_9_header_thunderbird=Who donates to Thunderbird?
+faq_item_9_paragraph_thunderbird=Anyone can donate, companies and individuals, to Thunderbird to protect the product now and in the future. 
 faq_item_10_header=Does my donation give me access to tech support?
 faq_item_10_paragraph=Anyone can access tech support for Mozilla’s products by visiting the <a href="https://support.mozilla.org/questions">support forum</a>. Mozilla does not provide tech support or enhanced tech support in exchange for donation.
+faq_item_10_paragraph_thunderbird=Anyone can access tech support for Thunderbird by visiting the <a href="https://support.mozilla.org/products/thunderbird">support forum</a>. Thunderbird does not provide tech support or enhanced tech support in exchange for donation.
 faq_item_tax_header=Is my donation tax deductible?
 faq_item_tax_paragraph_a=The Mozilla Foundation is a California non-profit corporation exempt from United States federal income taxation under IRC 501(c)3, and donations are tax deductible for United States persons (legal aliens and citizens). The Mozilla Foundation Tax ID is 20-0097189.
 faq_item_tax_paragraph_b=For donors outside of the United States, please consult with your tax adviser about whether your donation is tax deductible.
 faq_item_tax_paragraph_c=In the case of some substantial gifts, we may be required to report names and addresses to the IRS. Mozilla is required by law to report this information for its substantial donors and for cash equivalent in-kind gifts, like stocks and bitcoin, over $5,000.
 faq_item_11_header=How do I cancel or change my recurring donation?
 faq_item_11_paragraph=If you used PayPal to set up a recurring donation, you'll need to <a href="https://www.paypal.com/">log in to your PayPal account</a> to make changes or to cancel. If you made a monthly donation using credit card, please send an email to our support team at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>. Include the email address and name you used to make your donation and we'll try to help you within two business days.
+faq_item_11_paragraph_thunderbird=If you used PayPal to set up a recurring donation, you'll need to <a href="https://www.paypal.com/">log in to your PayPal account</a> to make changes or to cancel. If you made a monthly donation using credit card, please send an email to Mozilla's donations support team at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>. Include the email address and name you used to make your donation and they'll try to help you within two business days.
 faq_item_12_header=Can I donate in a currency other than United States dollars?
 faq_item_12_paragraph=Yes. On our <a href="https://donate.mozilla.org/">online form</a>, use the drop down menu on the form to select any of over two dozen currencies. We are able to accept paper checks in non-USD currencies, though they are more expensive for us to process.
```

Comment:
Please delete "We are able to accept paper checks in non-USD currencies, though they are more expensive for us to process." If it's not from Canada (which is a $5 cost), these checks cost us $45 to process, so I'd prefer not to mention donating by check in any other currency.

Reply:
Same here, that's just context showing, so let's do that change in another issue.